tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/test_eltwise_unary.py

```diff
@@ -648,6 +648,38 @@ def test_run_eltwise_heaviside(
             test_args,
         )
 
+    @skip_for_grayskull("#ToDo: GS implementation needs to be done for remainder op")
+    @pytest.mark.parametrize("value", [-0.5, -1, 0.5, 1, 2, 3, 4])
```

Comment:
I'd like this feature to also be tested with random values, not a set that is predetermined? Also extend the range 
```
        datagen_func = [
            generation_funcs.gen_func_with_cast(partial(generation_funcs.gen_rand, low=-100, high=100), torch.bfloat16)
        ]
```
to larger ranges

Reply:
Sure @rtawfik01 ,We have updated the range 

- (-1e5, 1e5) and 
- `test_args.update({"value": np.random.randint(-100, 100) + 0.5})`